tests/test.sh

```diff
@@ -1,3 +1,12 @@
-# Use "&&" so the test will fail if either python call fails. 
+# Run from top level directory of repository.
+
+# Copy over test data [takes about two minutes]
+# TODO: How will an automated CI tool access the E3SM data server? We could set up a public key for the CI server.
+# Change <username>
+scp -r <username>@blues.lcrc.anl.gov:/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_data tests/unit_test_data
```

Comment:
The test data directory is 274M in size. On VPN, the copying process took about 2 minutes. I imagine it will go faster if the tester isn't on VPN.